crane-cloud/backend
# Ft enable disable admin only

# Description

Enables Admin to enable and disable a project, and preventing users from enabling a project that is admin disabled

## Type of change

Please delete options that are not relevant.




## Trello Ticket ID

https://trello.com/c/pgLej1BC/1436-implement-project-enable-disable-for-admin-only

## How Can This Be Tested?

Use endpoint [/projects/{project_id}/admin_disable] to disable a user project and a user will not be able to enable it. Then use endpoint [/projects/{project_id}/admin_enable] to enable the project. Admin enable and disable should override any user enables or disables for the project.


## Checklist:
